src/strategy.coffee

```diff
@@ -37,7 +37,8 @@ class Strategy
 
     while tokenNumber <= tokensPositions.length - @minTokens
       mapFrame = currentMap.substring tokenNumber * 33, tokenNumber * 33 + @minTokens * 33
-      hash = crypto.createHash('md5').update(mapFrame).digest('hex').substring 0, 8
+      hash = crypto.createHash('md5').update(mapFrame).digest('hex')
```

Comment:
Crucial change here. I have a project large enough to guarantee collisions with this truncated hash, giving erroneous results.